### ISSUE TITLE
Why the Loss is negative?

### ISSUE THREAD

**Issue by bee992** [Clinical-and-Translational-Imaging-Lab/brats-e1d3#1]
I run the code and I find the loss is negative. Do you know why?

> Epoch 1/1000
--------------------

Batch:250/250.0: [########################################] 100%  Loss:-0.6802  Done...
loss/train:-0.6802 | dice_wt/train:0.7164 | dice_tc/train:0.5768 | dice_en/train:0.6219

**Comment by stalhabukhari** [comment]
Hi,

Sorry for the late reply. It is possible for the loss to be negative as we use `-SoftDice + CrossEntropy` loss. However, it will never be less than `-1`. Please refer to the accompanying paper for more detail.

Best,
Talha
